### DISCUSSION TITLE
Unable to connect to qualisys

### DISCUSSION THREAD

**Discussion by Trobrynn** [bitcraze/discussions#1159] [Q&A]
When I tried to connect to qualisys using the cfclient2018.12 version in Ubuntu 20, I couldn't connect and the terminal displayed the following message. 
`
Qobject::connect: cannot queue arguments of type 'OList<0PerststentModelIndex>(Make sure 'OList<OPersistentModelIndex>' is registered ustng qRegisterMetaTypelINFo:cflib.crazyflie,log:Have successfully started loggtng for id-1INFo:cflib.crazyflie,log:Have successfully started loggtng for id-3INFo:cflib.crazyflie,log:Have successfully started loggtng forINFo:cfclient.ui.tabs.locopositioning tab:No Loco deck installedINFo:cfclient.ui.tabs.FlightTab:[imu sensors.BMP388]: 1INFo:cfclient.ut tabs,FlightTab:[imu sensors.imuPhi]: 6.@INFo:cflib.crazyflie.log:Have successfully started logging for id-5INFo:cfclient.ul,tabs.FlightTab:[imu sensors.imuTheta]: 0.6INFo:cfclient.ui.tabs.FlightTab:[imu sensors.imuPsi]: @.@INFo:cfclient.ul.tabs.FlightTab:[imu sensors.AK8963]: 0INFo:cfclient.ui.tabs.FlightTab:[imusensors.LPs25H]: 0INF0:qtm:ConnectedERROR:qtm:<class 'Exception'>INFo:cfcllent.ui,tabs.QualisysTab:Failed to connectINFo:qtm:ConnectedERROR:qtm:<classException'>INFo:cfclient,ui,tabs,QualisysTab:Failed to connect
`

**Comment by knmcguire** [comment]
Hi! 

The qualisys tab doesn't exist in this version of the client so I'll convert this to a discussion.

**Comment by knmcguire** [comment]
So it seems that the QTM library can not connect to your qualisys system. Have you entered the right IP address? Is the computer streaming the rigid bodies?

I do think that this might be a better question for qualisys support since this [originated from their library](https://github.com/qualisys/qualisys_python_sdk). Have you tried asking them the question instead?
